src/components/bottom-sheet/BottomSheetContainer.tsx

```diff
@@ -1,24 +1,33 @@
 import React, {ReactNode} from 'react';
 import {useWindowDimensions, View} from 'react-native';
+import {BottomSheetHeader} from '@atb/components/bottom-sheet';
 
 export type BottomSheetContainerProps = {
   children: ReactNode;
   maxHeightValue?: number;
   fullHeight?: boolean;
   testID?: string;
+  bottomSheetTitle?: string;
+  closeBottomSheet?: () => void;
```

Comment:
Just call them `title` and `close`, it is self-explanatory that the params are related to bottom sheet since it is set on the `BottomSheetContainer`.